README.md

```diff
@@ -177,55 +145,33 @@ information about the bot, we encourage you to join our slack channel.
-[search our issue tracker](https://github.com/freqtrade/freqtrade/issues?q=is%3Aissue) 
-first. If it hasn't been reported, please 
-[create a new issue](https://github.com/freqtrade/freqtrade/issues/new) and 
-ensure you follow the template guide so that our team can assist you as 
+If you discover a bug in the bot, please
+[search our issue tracker](https://github.com/freqtrade/freqtrade/issues?q=is%3Aissue)
+first. If it hasn't been reported, please
+[create a new issue](https://github.com/freqtrade/freqtrade/issues/new) and
+ensure you follow the template guide so that our team can assist you as
 quickly as possible.
 
 ### [Feature Requests](https://github.com/freqtrade/freqtrade/labels/enhancement)
 
 Have you a great idea to improve the bot you want to share? Please,
 first search if this feature was not [already discussed](https://github.com/freqtrade/freqtrade/labels/enhancement).
-If it hasn't been requested, please 
-[create a new request](https://github.com/freqtrade/freqtrade/issues/new) 
-and ensure you follow the template guide so that it does not get lost 
+If it hasn't been requested, please
+[create a new request](https://github.com/freqtrade/freqtrade/issues/new)
+and ensure you follow the template guide so that it does not get lost
 in the bug reports.
 
 ### [Pull Requests](https://github.com/freqtrade/freqtrade/pulls)
 
 Feel like our bot is missing a feature? We welcome your pull requests!
 
-Please read our 
+Please read our
 [Contributing document](https://github.com/freqtrade/freqtrade/blob/develop/CONTRIBUTING.md)
-to understand the requirements before sending your pull-requests. 
+to understand the requirements before sending your pull-requests.
 
 Coding is not a neccessity to contribute - maybe start with improving our documentation?
 Issues labeled [good first issue](https://github.com/freqtrade/freqtrade/labels/good%20first%20issue) can be good first contributions, and will help get you familiar with the codebase.
 
 **Note** before starting any major new feature work, *please open an issue describing what you are planning to do* or talk to us on [Slack](https://join.slack.com/t/highfrequencybot/shared_invite/enQtMjQ5NTM0OTYzMzY3LWMxYzE3M2MxNDdjMGM3ZTYwNzFjMGIwZGRjNTc3ZGU3MGE3NzdmZGMwNmU3NDM5ZTNmM2Y3NjRiNzk4NmM4OGE). This will ensure that interested parties can give valuable feedback on the feature, and let others know that you are working on it.
 
-**Important:** Always create your PR against the `develop` branch, not `master`.
-
-## Requirements
-
-### Uptodate clock
-
-The clock must be accurate, syncronized to a NTP server very frequently to avoid problems with communication to the exchanges.
-
-### Min hardware required
-
-To run this bot we recommend you a cloud instance with a minimum of:
-
-- Minimal (advised) system requirements: 2GB RAM, 1GB disk space, 2vCPU
-
-### Software requirements
-
-- [Python 3.6.x](http://docs.python-guide.org/en/latest/starting/installation/) 
```

Comment:
I'm not certain if i wouldn't leave the requirements in the readme ... however if we add a link to the documentation section of that would suffice too i think.

Reply:
Put the requirements back to the hood